src/components/Pickers/BasePicker.Props.ts

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import { IPickerItemProps } from './PickerItem.Props';
+
+export interface IBasePickerProps<T> extends React.Props<any> {
+  /**
+   * Function that specifies how the selected item will appear.
+   */
+  onRenderItem?: (props: IPickerItemProps<T>) => JSX.Element;
+  /**
+   * Function that specifies how the suggestion will appear
+   */
+  onRenderSuggestion?: (props: T) => JSX.Element;
+  /**
+   * A callback for what should happen when a person types text into the input.
+   * Returns the already selected items so the resolver can filter them out.
+   */
+  onResolveSuggestions: (filter: string, selectedItems?: T[]) => any[];
+  /**
+   * Initial items that have already been selected and should appear in the people picker.
+   */
+  startingItems?: T[];
```

Comment:
Maybe use "defaultSelectedItems". (Is more consistent with controlled component intialization examples like "defaultChecked" and "defaultValues"